tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -27,10 +27,14 @@ LIBC=${LIBC:-musl}
 # However, it is not enforced by default: you need to enable that in the main configuration file.
 SECCOMP=${SECCOMP:-"yes"}
 SELINUX=${SELINUX:-"no"}
+AGENT_POLICY=${AGENT_POLICY:-no}
+AGENT_POLICY_DEBUG=${AGENT_POLICY_DEBUG:-no}
```

Comment:
Are we setting these variables in the GHA job?

Reply:
What is the GHA job?

If you are referring to adding test coverage for policy for non-default configurations: I plan to do that next.

Reply:
We're not, I've discussed with @danmihai1 what's the way to approach with the tests, and we agreed to have the tests coming in a different PR.

This PR will ensure that no regressions are caused, the second PR will enable this for the guest images we care about, and add the tests for those.

Reply:
BTW, I don't think we actually need to set anything in the GHA.
Adding an `export AGENT_POLICY=yes` to https://github.com/kata-containers/kata-containers/blob/7bf994827daf307b4cceea099cf8aff8a4d400e5/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L186-L189 was what we need to get it built by default for the TDX image.

This should **NOT** be done as part of this PR, though, and the message is a JFYI kind of comment,

Reply:
> What is the GHA job?

GHA = GitHub Actions